src/App.js

```diff
@@ -1,14 +1,78 @@
-import './App.css';
-import Education from './components/Education';
-import Experience from './components/Experience';
-import General from './components/General';
+import { useState } from 'react';
+import styles from './app.module.css';
+import { Education, Experience, General } from './components/';
+
+const initialData = {
+  general: { name: 'John', email: 'john@john.john', phone: '+334455667788' },
+  education: [
+    {
+      school: 'Very Nice school',
+      title: 'Pupil',
+      date: { from: '2000-06', to: '2003-07' },
+    },
+    {
+      school: 'University',
+      title: 'Student',
+      date: { from: '2004-06', to: '2009-07' },
+    },
+  ],
+  experience: [
+    {
+      company: 'BBBB',
+      position: 'Worker',
+      description: 'nice',
+      date: { from: '2010-01', to: '2015-02' },
+    },
+  ],
+};
+
 function App() {
+  const [data, setData] = useState(initialData);
```

Comment:
`<App /> ` component is a parent component of the whole project, therefore it should not contain any logic in itself. Usually you would use <App /> to wrap Context, Router or any other top level stuff. In your case your App should look something like `return <div className="app"><CvPage /></div>` and that CvPage would have all that logic.